metrics/internal/collectors/ceph-block-pool_test.go

```diff
@@ -0,0 +1,324 @@
+package collectors
+
+import (
+	"testing"
+
+	"github.com/openshift/ocs-operator/metrics/internal/options"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	cephv1listers "github.com/rook/rook/pkg/client/listers/ceph.rook.io/v1"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
```

Comment:
Instead of duplicating everything, can you reuse existing mocks and utility functions?
Maybe refactor it out to a common file that all test files can use?

Reply:
ack